java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -914,6 +914,13 @@ private String getBlockName(Block b) {
         if (b != null) {
             String sName = b.getSystemName();
             String uName = b.getUserName();
+            // Jay Janzen
```

Comment:
Why is the Block's User Name inadequate for block identification? We provide user names precisely for this reason (to provide a name for the block that is meaningful for people, not computers).